Comments/ClientApp/src/document-processing/transforms/types.js

```diff
@@ -11,7 +11,7 @@ export function nodeIsInternalLink(node) {
 }
 
 export function nodeIsSubsection(node) {
-	return node.name === "p" && node.attribs && node.attribs["data-heading-type"] === "numbered-paragraph";
+	return (node.name === "p" || node.name === "article") && node.attribs && node.attribs["data-heading-type"] === "numbered-paragraph";
```

Comment:
small point but would optional chaining be better, e.g. node.attribs?.["data-heading-type"] - although the long winded version is used everywhere else so probably not worth it.